lib/util.js

```diff
@@ -497,6 +497,14 @@ function getConstructorName(obj) {
 }
 
 function getPrefix(constructor, tag, fallback) {
+
+  if (constructor === '') {
+    if (tag !== '' && constructor !== tag) {
+      return `[${fallback}: null prototype] [${tag}] `;
+    }
+    return `[${fallback}: null prototype] `;
+  }
+
   if (constructor !== '') {
```

Comment:
This condition tests for `constructor !== ''` so instead of testing in another if for the other case, we can just move the code above below this if. In that case we already know, that the constructor is an empty string and don't have to check again.

Reply:
Well this is true, but the line:

```
return `[${fallback}: null prototype] `;
```

definitely needs a check whether a  `constructor` is empty or not.

Reply:
What I mean is:

```js
if (constructor !== '') {
  ... as before ...
}
if (tag !== '' && fallback !== 'tag') {
  return `[${fallback}: null prototype] [${tag}] `;
} 
return `[${fallback}: null prototype] `;
```

That way we are sure the constructor is definitely empty.